test/integration/referral.spec.ts

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { ReferralError } from '../../src/ports/referral/types'
+import { test } from '../components'
+
+test('Referral', ({ components }) => {
```

Comment:
As this is just a component, I wouldn't test is as an integration. What do you think about using a normal describe here?